### DISCUSSION TITLE
为有潜在价值的参考引用建立一个阅读收件箱

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#62] [主发布站的评论系统]
# 为有潜在价值的参考引用建立一个阅读收件箱

%%我的理解:: 那些不需要做笔记的阅读内容，不应该成为真正的稍后读。我们需要的是一个有质量的稍后读inbox，其中的每天文章都要写点笔记。这样的阅读有闭环，才是有效的阅读。%%
浏览器标签逐渐堆积的，PDF 文件夹不断激增，这一切真让人焦虑。为了避免这一点，重要的是要有一个程序，来自动化地保存阅读中那些有潜在价值的参考材料。
一旦捕获材料，收集箱中每个项目要么：
 删除（最后看起来不值得详细阅读） 被认真阅读（即[[写你所读，以求内化]]) 被浅读，并在参考书库中存档
4.（也许）被添加到其他一些清单中，如「待烹调的食谱」  重点在于，这不是一个「也许有一天会读」的清单。它不会无限期地积累，因为那样的话，它就不是一个可靠的方法来[[闭合开环]]。
因此，在构建一个阅读收件箱时，重要的考虑因素是：
 书籍、文章、网页的零阻力捕获（以轻松地关闭该循环） 零阻力地查看收件箱项目对应的阅读量 零阻力地跨类型列举（译注：作为清单或者引用） 收件箱应鼓励删除滞留的项目（例如，当一个项目被多次跳过时，应该很明显。）  有趣的是，现有的「稍后阅读」或参考管理系统都不符合这些标准。它们通常是按内容类型划分的，而且没有一个系统鼓励删除徘徊的项目。另见：[[谨防自动导入阅读收件箱]]。

https://garden.oldwinter.top/%E4%B8%BA%E6%9C%89%E6%BD%9C%E5%9C%A8%E4%BB%B7%E5%80%BC%E7%9A%84%E5%8F%82%E8%80%83%E5%BC%95%E7%94%A8%E5%BB%BA%E7%AB%8B%E4%B8%80%E4%B8%AA%E9%98%85%E8%AF%BB%E6%94%B6%E4%BB%B6%E7%AE%B1

**Comment by j1g5awi** [comment]
> 1. 删除（最后看起来不值得详细阅读）
  > 2. 被认真阅读（即写你所读，以求内化)
  > 3. 被浅读，并在参考书库中存档
  > 4.（也许）被添加到其他一些清单中，如「待烹调的食谱」

此处 `4.` 后少了个空格。

**Comment by oldwinter** [comment]
谢谢你的仔细。已修改啦~~

Jigsaw ***@***.***> 于2023年3月24日周五 09:55写道：
